proto/cosmwasm/wasm/v1beta1/proposal.proto

```diff
@@ -0,0 +1,121 @@
+syntax = "proto3";
+package cosmwasm.wasm.v1beta1;
```

Comment:
These are just moved around, right?

Seems like cleaner path naming

Reply:
ja, files were just moved und updates on the path for imports.